src/main/webapp/js/instructorFeedbackResultsAjaxByQuestion.js

```diff
@@ -49,6 +49,10 @@ $(document).ready(function() {
                 $(panelHeading).off('click');
                 $(panelHeading).click(toggleSingleCollapse);
                 $(panelHeading).trigger('click');
+                
+                if ($(panelBody[0]).find('.no-response').length !== 0) {
+                    $(panelBody[0]).parent().parent().removeClass('panel-info').addClass('panel-default');
+                }
```

Comment:
A one line comment explaining what this `if` clause does can be useful. Or extract into a separate method with a self-explanatory name.